DepositeCalcTests/Tests/CalculatorPageTests.cs

```diff
@@ -0,0 +1,268 @@
+        }
+
+        [TestCase(1, 1, 1)]
+        [TestCase(10, 10, 1)]
+        [TestCase(99000, 10, 1)]
+        [TestCase(100000, 10, 1)]
+        [TestCase(10, 99, 1)]
+        [TestCase(10, 100, 1)]
+        [TestCase(10, 50, 364)]
+        [TestCase(10, 50, 365)]
+        public void ValidCalculation365Test(double depositAmount, double interestRate, double investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.ValidCalculation(depositAmount, interestRate, investmentTerm);
+            calculatorPage.ClickOnFinancialYear365RadioBtn();
+            calculatorPage.ClickOnCalculateBtn();
+
+            double oneDayPercentageCalculation = (depositAmount * (interestRate * 0.01) / 365) * investmentTerm;
+            double incomeCalculation = depositAmount + oneDayPercentageCalculation;
+
+            string expectedInterestEarned = string.Format("{0:0.00}", oneDayPercentageCalculation);
+            string expectedIncome = string.Format("{0:0.00}", incomeCalculation);
+
+            // Asserts
+            Assert.AreEqual(expectedInterestEarned, calculatorPage.GetInterestEarnedFldValue(), "Incorrect value in the Interest earned field");
+            Assert.AreEqual(expectedIncome, calculatorPage.GetIncomeFldValue(), "Incorrect value in the Income field");
+        }
+
+        [TestCase(1, 1, 1)]
+        [TestCase(10, 10, 1)]
+        [TestCase(99000, 10, 1)]
+        [TestCase(100000, 10, 1)]
+        [TestCase(10, 99, 1)]
+        [TestCase(10, 100, 1)]
+        [TestCase(10, 50, 359)]
+        [TestCase(10, 50, 360)]
+        public void ValidCalculation360Test(double depositAmount, double interestRate, double investmentTerm)
+        {
+            // Arrange
+            var calculatorPage = new CalculatorPage(driver);
+
+            // Act
+            calculatorPage.ValidCalculation(depositAmount, interestRate, investmentTerm);
+            calculatorPage.ClickOnFinancialYear360RadioBtn();
+            calculatorPage.ClickOnCalculateBtn();
+
+            double oneDayPercentageCalculation = (depositAmount * (interestRate * 0.01) / 360) * investmentTerm;
+            double incomeCalculation = depositAmount + oneDayPercentageCalculation;
+
+            string expectedInterestEarned = string.Format("{0:0.00}", oneDayPercentageCalculation);
+            string expectedIncome = string.Format("{0:0.00}", incomeCalculation);
+
+            // Asserts
+            Assert.AreEqual(expectedInterestEarned, calculatorPage.GetInterestEarnedFldValue(), "Incorrect value in the Interest earned field");
+            Assert.AreEqual(expectedIncome, calculatorPage.GetIncomeFldValue(), "Incorrect value in the Income field");
+        }
+
+        [TestCase(100001, 1, 1)]
+        [TestCase(100002, 1, 1)]
+        [TestCase(10, 101, 1)]
+        [TestCase(10, 102, 1)]
+        [TestCase(10, 1, 366)]
+        [TestCase(10, 1, 367)]
```

Comment:
These tests should be enough:
```suggestion
        [TestCase(100001, 1, 1)]
        [TestCase(10, 101, 1)]
        [TestCase(10, 1, 366)]
```